### PR TITLE
[AzureStack] Bug Fixes, updated Release notes, cmdlet rename

### PR DESCRIPTION
**Pull request by deathly809**
## Description

1. Bug fix for a bug fix (Page-able)
2. We were overriding ErrorActionPreference, removed this override.
3. Updated KeyVault version (module and AzureStack) because of (2)
4. Fixed a bug in Azs.Subscription.Admin where if a user provided a resource group name we still ask for it.
5. Updated release notes to reflect the above.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by deathly809** [comment]
PS-Sign job here,

https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/983/

File: src/StackAdmin/Azs.Backup.Admin/Module/Azs.Backup.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsEncryptionKeyBase64.ps1 [hunk 1]
```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License. See License.txt in the project root for license
     Generate encryption key for infrastructure backups.
 
 #>
-function New-EncryptionKeyBase64
```
Comment (bganapa) [inline src/StackAdmin/Azs.Backup.Admin/Module/Azs.Backup.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsEncryptionKeyBase64.ps1]: Pls rename the example as well. Mention the parent cmdlet where this helper cmdlet would be used in the description

File: src/StackAdmin/Azs.Network.Admin/Module/Azs.Network.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsNetworkQuota.ps1 [hunk 2]
```diff
@@ -108,7 +108,7 @@ function New-AzsNetworkQuota {
 
     Process {
 
-        $ErrorActionPreference = 'Stop'
+
```
Comment (bganapa) [inline src/StackAdmin/Azs.Network.Admin/Module/Azs.Network.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsNetworkQuota.ps1]: Can you please clear this extra whitespace everywhere?

Comment (deathly809) [reply src/StackAdmin/Azs.Network.Admin/Module/Azs.Network.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsNetworkQuota.ps1]: Why?  I would have to edit 143+ files to make three blank lines two blank lines.

File: src/StackAdmin/Azs.Backup.Admin/Module/Azs.Backup.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsEncryptionKeyBase64.ps1 (hunk 1 above)
Comment (deathly809) [reply src/StackAdmin/Azs.Backup.Admin/Module/Azs.Backup.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/New-AzsEncryptionKeyBase64.ps1]: Done.

**Comment by bganapa** [comment]
Another ps-sign running here https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/994/

File: src/StackAdmin/Azs.KeyVault.Admin/Module/Azs.KeyVault.Admin/Azs.Keyvault.Admin.psd1 [hunk 3]
```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License. See License.txt in the project root for license
     RootModule        = 'Azs.KeyVault.Admin.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '0.1.0'
```
Comment (markcowl) [inline src/StackAdmin/Azs.KeyVault.Admin/Module/Azs.KeyVault.Admin/Azs.Keyvault.Admin.psd1]: The version is updated here, does it need to be updated for the other modules where you made changes?

Comment (bganapa) [reply src/StackAdmin/Azs.KeyVault.Admin/Module/Azs.KeyVault.Admin/Azs.Keyvault.Admin.psd1]: The other module versions are already updated with previous PRs

**Pull request merged**